recipes/snpeff/meta.yaml

```diff
@@ -22,7 +22,7 @@ source:
 
 requirements:
   run:
-    - openjdk
+    - openjdk=8.0.192
```

Comment:
@cbrueffer wrote:
>SnpEff works fine here with OpenJDK 11. Have you tried from a fresh Miniconda installation, or an environment?

At the very least, don't pin to a specific version, but to a range, e.g. `>=8`.